ja/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -0,0 +1,285 @@
+**1. Machine Learning tips and tricks cheatsheet**
+
+&#10230;　機械学習チップ&トリック　チートシート
```

Comment:
```suggestion
&#10230;機械学習のアドバイスやコツのチートシート
```
To be consistent with translation of "Deep learning tips and tricks cheatsheet":
https://stanford.edu/~shervine/l/ja/teaching/cs-230/cheatsheet-deep-learning-tips-and-tricks